### ISSUE TITLE
hangs on air gapped environment 

### ISSUE THREAD

**Issue by sionsmith** [confluentinc/cli#1697]
Since V3 of the CLI has been released everything I try and execute hangs, even running -h does this. I have showed the full output below. I have also update the config with:
```
  "version": "1.0.0",
  "disable_update_check": true,
  "disable_updates": true,
  "disable_plugins": true,
```

*output*
```
[root@apptuv72fd ~]# /opt/app/software/confluent-7.3.1/bin/confluent  login --url https://xxxxx.xxxx.test.group:8090  -vvvvv

2023-02-08T13:28:55.939Z [DEBUG] Did not find full credential set from environment variables
2023-02-08T13:28:55.940Z [DEBUG] Searching for netrc machine with filter: {IgnoreCert:true IsCloud:false IsSSO:false Name: URL:https://xxxx.xxxxx.cloud.test.group:8090}
2023-02-08T13:28:55.940Z [DEBUG] Get netrc machine error: open /root/.netrc: no such file or directory
```

**Comment by sionsmith** [comment]
After further investigation I have the following:
```
/opt/app/software/confluent-7.3.1/bin/confluent --version                                              
confluent version v2.28.1
```

This took around about 5 minutes to return. Any ideas about what could be causing this? I have another versioned installed:
```
[root@apptuv72fd ~]# /opt/app/software/confluent-7.2.1/bin/confluent --version
confluent version v2.19.0
```

This returns immediately with the configuration file as above

**Comment by brianstrauch** [comment]
We ran into a similar issue a couple months ago where the CLI was trying to fetch feature flags and timing out. It sounds like your specific issue has been fixed and will be included with CP 7.4.

In the meantime, you can try [downloading the latest independently released version of the CLI](https://github.com/confluentinc/cli#installation).
